### ISSUE TITLE
SEO

### ISSUE THREAD

**Issue by kenobyzy** [jevlabs/website#7]
- [x]  Add pictures, titles, descriptions from md files using yaml front matter (partially implemented) and have fallbacks
- [x]  Add optimized favicons
- [x]  Rethink headings and text structure
- [x]  Fix accessibility issues (missing alt texts,...)
